apps/dotcms-block-editor/src/app/dot-block-editor/dot-block-editor.component.scss

```diff
@@ -11,9 +11,8 @@ dotcms-block-editor {
         -moz-box-sizing: border-box;
         border: solid 2px #222;
         display: block;
-        height: 100%;
```

Comment:
Why?

Reply:
because of this. 
![image](https://user-images.githubusercontent.com/31667212/142498326-519ef481-848d-4ba0-a42a-3a4d254cda4c.png)

Reply:
put it back after conversation